### DISCUSSION TITLE
Pinia adds its own properties to the store?

### DISCUSSION THREAD

**Discussion by eric-2d** [vuejs/pinia#1005] [Help and Questions] [answered]
I recently switched from vuex and was migrating along just fine until I reached my final step.
Which is persisting a store to my SQl db, literally `axios.post(resource, this.store)`
My DB rejected the insert because of `$id` and `_isOptionsAPI` columns don't exist in my table

I understand pinia might need those internally, but the fact that they are added without my knowledge feels like it's polluting the state.

What am I to do? simply `delete store.$id` and `delete store._isOptionsAPI` ?
What if other internal variables are introduced and break my app?

**Comment by angelhdzmultimedia** [comment, marked as answer]
```js
 axios.post(resource, this.store.$state)
```

**Reply by eric-2d** [reply]
Cheers
